spec/defines/interface_spec.rb

```diff
@@ -40,6 +40,38 @@
         it { is_expected.to contain_file("/etc/systemd/network/#{title}.network").without_content(%r{MTUBytes}) }
         it { is_expected.not_to contain_ferm__rule("allow_wg_#{title}") }
       end
+      context 'with required params (peers) and without firewall rules' do
+        let :params do
+          {
+            peers: [
+              {
+                public_key: 'blabla==',
+                endpoint: 'wireguard.example.com:1234',
+              }
+            ],
```

Comment:
Maybe we should add at least two peers here just to test that the file is rendered correctly with multiple peers?

Reply:
added a second one